python/graphstorm/model/rgcn_encoder.py

```diff
@@ -327,12 +370,21 @@ class RelationalGCNEncoder(GraphConvEncoder, GSgnnGNNEncoderInterface):
         model.set_decoder(EntityClassifier(model.gnn_encoder.out_dims, 3, False))
 
         h = do_full_graph_inference(model, np_data)
+
+   .. warning::
+
+        To use edge feature in message passing computation, please ensure the node and edge
+        features have the same dimension. Users can use GraphStorm's ``GSNodeEncoderInputLayer``,
+        and ``GSEdgeEncoderInputLayer`` to transfer node and edge feature dimensions.
+
     """
     def __init__(self,
                  g,
                  h_dim, out_dim,
                  num_bases=-1,
                  num_hidden_layers=1,
+                 edge_feat_name=None,
```

Comment:
Shall we add these two arguments in GraphConvEncoder abs class?

Reply:
Add them to the `GraphConvEncoder` and check if children classes will support edge features if set `edge_feat_name` is not None.